core/field_dropdown.ts

```diff
@@ -334,7 +334,7 @@ export class FieldDropdown extends Field<string> {
 
       const [label, value] = option;
       const content = (() => {
-        if (typeof label === 'object') {
+        if (isImageProperties(label)) {
           // Convert ImageProperties to an HTMLImageElement.
           const image = new Image(label['width'], label['height']);
```

Comment:
do you want to update these to use `.` access while you're here and updating other uses?

Reply:
Good call, done!